testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -143,6 +143,107 @@ QUnit.module("common", {}, () => {
         assert.equal(valueChangedHandler.callCount, 1, "valueChangedHandler has been called");
         assert.equal(valueChangedHandler.getCall(0).args[0].event.type, "dxclick", "event is correct");
     });
+
+    QUnit.test("T810808 - should be possible to type characters on android 2.3 and 4.1 in TextBox with maxLength and mask", assert => {
+        const originalDevices = devices.real();
```

Comment:
We do [not support](https://js.devexpress.com/Documentation/18_1/Guide/Common/Supported_Browsers/) integrated browser for android version < 4.4 since 18.1